src/components/Button/Button.stories.mdx

```diff
@@ -191,12 +195,17 @@ never to link and go to another page. For `Button`s without text and only an
 icon, make sure to pass an `aria-label` value to the `Button`. `Button`s cannot
 have images of text.
 
+`Button`s can be rendered in three sizes through the `size` prop: "small",
```

Comment:
put prop values in back ticks for consistency with the component props , i.e. `"small"`